### PR TITLE
Feature/789 add raster to polygon to vessel detection

### PR DESCRIPTION
**Pull request by rasrocha**
Manifests for the Raster-to-Polygon and Speckle Filter tools have been created with the respective run files. All tools can be successfully packed and shipped to WINGS.

### CODE REVIEW

File: src/coresyf_rasterToPolygon/examples.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+ASA_WSM_1PNUPA20050206_102659_000000672034_00280_15364_1195.N1.zip="http://example.org"
```
Comment (rmcapote) [inline src/coresyf_rasterToPolygon/examples.sh]: Change this line to:
multiBandImage.tif="http://example.org"

The behavior is: 
1. The first line of examples.sh is read (mandatory line).
2. The file is downloaded from the provided URL if it doesn't already exists on local directory.
3. It runs the command line provided in the last line.

File: src/coresyf_rasterToPolygon/rasterToPolygon.manifest.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{
+    "name": "CoReSyF RasterToPolygon",
+    "type": "RasterToPolygon",
+    "description": "CoReSyF tool to convert a Raster to a Shapefile",
```
Comment (rmcapote) [inline src/coresyf_rasterToPolygon/rasterToPolygon.manifest.json:4]: Perhaps we can complete the description by adding the following text:
"It creates a shapefile with a layer containing polygon features for all connected regions of pixels in the raster sharing a common pixel value. Each polygon is created with an attribute indicating the pixel value of that polygon. The shapefile contains a layer with its own name, but the user can specify the fieldname for the pixel values."

File: src/coresyf_rasterToPolygon/rasterToPolygon.manifest.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+            "name": "source",
+            "description": "Sets source to <filepath>",
+            "type": "Raster"
+        }
+    ],
+    "parameters": [
+        {
+            "identifier": "band",
+            "name": "band",
+            "description": "Indicate the source band to process",
```
Comment (rmcapote) [inline src/coresyf_rasterToPolygon/rasterToPolygon.manifest.json]: Perhaps change the description to:
"Indicate the band on the raster file to build the polygons features from."

File: src/coresyf_speckle_filter/CoReSyFSpeckleFilter.manifest.json [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+            "description": "Sets source 'source' to <filepath>",
+            "type": "Raster"
+        }
+
+    ],
+    "parameters": [
+        {
+            "identifier": "Pfilter",
+            "name": "Filter",
+            "description": "Value must be one of 'None', 'Boxcar', 'Median', 'Frost', 'Gamma Map', 'Lee', 'Refined Lee', 'Lee Sigma', 'IDAN'",
```
Comment (91nunocosta) [inline src/coresyf_speckle_filter/CoReSyFSpeckleFilter.manifest.json]: The `options` field should be use to specify the allowed values.

File: src/coresyf_speckle_filter/examples.sh [hunk 5]
```diff
@@ -0,0 +1,4 @@
+#ASA_WSM_1PXPDE20021117_104431_000000672011_00180_03741_0009.N1.gz="http://example.org"
```
Comment (91nunocosta) [inline src/coresyf_speckle_filter/examples.sh]: Are the tests passing with this line commented out?

**Review by 91nunocosta — COMMENTED** [review]
We should not forget to specify the parameter options in manifests. But for now is ok.
For me, If all the tests are passing this branch can be merged.

File: src/coresyf_speckle_filter/CoReSyFSpeckleFilter.manifest.json [hunk 6]
```diff
@@ -0,0 +1,46 @@
+{
+    "name": "CoReSyFSpeckleFilter",
+    "type": "SpeckleFilter",
+    "operation":  {"operation": "Speckle-Filter"},
+    "description": "Speckle Reduction",
```
Comment (rmcapote) [inline src/coresyf_speckle_filter/CoReSyFSpeckleFilter.manifest.json]: "Speckle Reduction from SNAP GPT command line tool"

File: src/coresyf_speckle_filter/examples.sh (hunk 5 above)
Comment (rmcapote) [inline src/coresyf_speckle_filter/examples.sh]: Is this working with this line as comment?

**Review by rmcapote — CHANGES_REQUESTED** [review]
Please perform the suggested corrections.

File: src/coresyf_rasterToPolygon/rasterToPolygon.manifest.json (hunk 2 above)
Comment (rasrocha) [reply src/coresyf_rasterToPolygon/rasterToPolygon.manifest.json:4]: added

File: src/coresyf_rasterToPolygon/rasterToPolygon.manifest.json (hunk 3 above)
Comment (rasrocha) [reply src/coresyf_rasterToPolygon/rasterToPolygon.manifest.json]: done

File: src/coresyf_speckle_filter/CoReSyFSpeckleFilter.manifest.json (hunk 6 above)
Comment (rasrocha) [reply src/coresyf_speckle_filter/CoReSyFSpeckleFilter.manifest.json]: done

File: src/coresyf_speckle_filter/examples.sh (hunk 5 above)
Comment (rasrocha) [reply src/coresyf_speckle_filter/examples.sh]: I uncommented the line and changed it to match the name of the input.

**Pull request merged**